source/shared/net/resources/dashboards.rst

```diff
@@ -0,0 +1,107 @@
+.. _dashboards:
+
+===========================
+Dashboards and Status Pages
+===========================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+Dashboard pages
+===============
+
+To open the dashboards, click the link for the  `**Mainnet dashboard and block explorer** <https://dashboard.mainnet.concordium.software/>`_ or `**Testnet dashboard and block explorer** <https://dashboard.testnet.concordium.software/>`_.
```

Comment:
```suggestion
To open the dashboards, click the link for the  `Mainnet dashboard and block explorer <https://dashboard.mainnet.concordium.software/>`_ or `Testnet dashboard and block explorer <https://dashboard.testnet.concordium.com/>`_.
```

Something was wrong with the bold markers, and the Testnet dashboard is on a .com domain for some reason.

![image](https://user-images.githubusercontent.com/76045015/146524234-09a26f48-f0ad-4b35-a3f9-f054a1dc9d8f.png)